R/importData.R

```diff
@@ -42,7 +42,7 @@ importData <- function(subtype) {
   # SOURCE DIRECTORY -----------------------------------------------------------
 
   # check if source folder exists
-  sourceDir <- getConfig("sourcefolder")
+  sourceDir <- suppressMessages(getConfig("sourcefolder"))
```

Comment:
As I don't get a message with the old line I'm wondering if suppressing it is wise. What message did you get?

Reply:
When not suppressing the output of `getConfig`, my err log file contains the definition of the `madrat` default configuration such as: 
```
Initializing madrat config with default settings
regionmapping = regionmappingH12.csv
     extramappings = NULL
     packages = madrat
     globalenv = FALSE
     verbosity = 1
     mainfolder = /p/projects/rd3mod/inputdata
     sourcefolder = NA
     cachefolder = /p/projects/rd3mod/inputdata/cache_1.27
     mappingfolder = NA
     outputfolder = /p/projects/rd3mod/inputdata/output_1.27
     pucfolder = /p/projects/rd3mod/inputdata/puc_1.27
     tmpfolder = NA
     nolabels = NULL
     forcecache = FALSE
     ignorecache = NULL
     cachecompression = gzip
     hash = xxhash32
     diagnostics = FALSE
     debug = FALSE
     maxLengthLogMessage = 200
  ..done!
```
I was not sure whether this would be needed at all.

Reply:
I think you should be able to solve it this way
```suggestion
  sourceDir <- getConfig("sourcefolder", verbose = FALSE)
```